angel-data/onboarding-NO-v2.json

```
@@ -590,7 +579,7 @@
       "selected": false,
       "story": "ecb9a7d2-1aa1-448c-be68-2b4ecabf948a",
       "tags": [],
-      "text": "<Message>\n  {NO_SIGNUP_EMAIL_MESSAGE_LAST}\n</Message>\n\n<TextAction placeholder=\"{EMBARK_EMAIL_PLACEHOLDER}\"\n            mask=\"Email\"\n            key=\"email\"\n            next=\"[[hidden:{EMBARK_CONTINUE}->OfferSelector]]\"\n>\n</TextAction>",
+      "text": "<Message>\n  {NO_SIGNUP_EMAIL_MESSAGE_LAST}\n</Message>\n\n<TextAction placeholder=\"{EMBARK_EMAIL_PLACEHOLDER}\"\n            mask=\"Email\"\n            key=\"email\"\n            next=\"[[{EMBARK_CONTINUE}->OfferSelector]]\"\n>\n</TextAction>\n§",
```

Do Angel usually add theses type of characters? 🤔 

```suggestion
      "text": "<Message>\n  {NO_SIGNUP_EMAIL_MESSAGE_LAST}\n</Message>\n\n<TextAction placeholder=\"{EMBARK_EMAIL_PLACEHOLDER}\"\n            mask=\"Email\"\n            key=\"email\"\n            next=\"[[{EMBARK_CONTINUE}->OfferSelector]]\"\n>\n</TextAction>\n", 
```

---

No, I believe it was a typo from my side. Great catch btw